src/docs/patterns/visualizations.mdx

```diff
@@ -122,18 +122,15 @@ There are not as many charting libraries available for React Native applications
 
 There are many libraries and services for data visualizations. Brightlayer UI currently provides themes / utilities for the following libraries:
 
-**Angular** and **React**:
+**React**:
 
--   Charts & Graphs
+-   Brightlayer Charts
```

Comment:
Combine all of this into one line like:

'- [Brightlayer Charts](link to docs) ([Storybook](link to storybook))'

Reply:
This section isn't really about explaining what these recommendations are, just linking to them. We will revamp this entire page in a future story to better steer users in that direction. For now we're just trying to remove references to deprecated stuff.